app/src/main/res/layout/fragment_settings.xml

```diff
@@ -1,250 +1,405 @@
-        android:layout_marginTop="10dp"
-        android:backgroundTint="#00ffffff"
-        android:src="@drawable/ask_question_button"
-        app:layout_constraintEnd_toEndOf="@id/rectangleSettings"
-        app:layout_constraintStart_toStartOf="@id/rectangleSettings"
-        app:layout_constraintTop_toBottomOf="@id/supportBlockTitle"
-        android:contentDescription="@string/todo" />
-
-    <ImageButton
-        android:id="@+id/contactSupport"
-        android:layout_width="wrap_content"
-        android:layout_height="48dp"
-        android:backgroundTint="#00ffffff"
-        android:src="@drawable/contact_support_button"
-        app:layout_constraintEnd_toEndOf="@id/rectangleSettings"
-        app:layout_constraintStart_toStartOf="@id/rectangleSettings"
-        app:layout_constraintTop_toBottomOf="@id/askQuestion"
-        android:contentDescription="@string/todo" />
-
-    <ImageButton
-        android:id="@+id/review"
-        android:layout_width="wrap_content"
-        android:layout_height="48dp"
-        android:backgroundTint="#00ffffff"
-        android:src="@drawable/review_button"
-        app:layout_constraintEnd_toEndOf="@id/rectangleSettings"
-        app:layout_constraintStart_toStartOf="@id/rectangleSettings"
-        app:layout_constraintTop_toBottomOf="@id/contactSupport"
-        android:contentDescription="@string/todo" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        android:layout_margin="16dp"
+        >
+
+        <TextView
+            android:id="@+id/textSettings"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:fontFamily="@font/inter_bold"
+            android:text="@string/title_settings"
+            android:textColor="@color/grey"
+            android:textSize="36sp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/rectangleProfile"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/textSettings">
+
+            <View
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                app:layout_constraintTop_toTopOf="@id/rectangleProfile"
+                app:layout_constraintBottom_toBottomOf="@id/rectangleProfile"
+                app:layout_constraintStart_toStartOf="@id/rectangleProfile"
+                app:layout_constraintEnd_toEndOf="@id/rectangleProfile"
+                android:background="@drawable/rounded_rectangle_grey"
+                />
+
+            <de.hdodenhof.circleimageview.CircleImageView
```

Comment:
We should not use the final values of the button size parameters. Need fix it in android:id="@+id/userAvatar". But we can try to merge the branches, and then see how it will look and, if anything, fix it.

Reply:
Do you mean that we shouldn't use values like, for example, "20dp"?